finder/index.html

```diff
@@ -0,0 +1,224 @@
+                    <p>Fetching Repos</p>
+                </div>
+        </section>
+
+        <section id="results" style="display:none">
+            <table class="table is-striped is-hoverable" id="table-projects">
+                <thead>
+                    <tr>
+                    <th>Name</th>
+                    <th>Description</th>
+                    <th>License</th>
+                    <th>Stars</th>
+                    <th>Url</th>
+                    <th>Updated At</th>
+                    </tr>
+                </thead>
+                <tfoot>
+                        <tr>
+                            <th>Total Repositories Found</th>
+                            <th id="total_count"></th>
+                        </tr>
+                        </tfoot>
+                <tbody id="table-projects-body">
+                    <!-- Repos here -->
+                </tbody>
+            </table>
+        </section>
+
+        
+        
+        <div class="margin-bottom-50px"></div>
+    </main>
+
+    <footer>
+        <section class="columns is-centered" id="bottom">
+            <div class="column is-half has-text-centered">
+                <a href="https://github.com/vaibhavmule/awesome-masonite">Awesome Masonite</a>
+            </div>
+        </section>
+        <div class="margin-bottom-50px"></div>
+    </footer>
+
+    <!-- Scripts -->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.22.2/moment.min.js"></script>
+
+    <!-- Fetch Repos -->
+    <script type="text/javascript">
+        
+        let results = {
+            total: 0,
+            items: []
+        }
+
+        $(document).ready(() => {
+            let endpoint = `https://api.github.com/search/repositories?q=masonite&topic=masonite&sort=stars&order=desc`
+            
+            console.log('Fetching', endpoint)
+            
+            $.ajax({
+                url:endpoint,
+                type: 'GET',
+                headers: {'Accept' : 'application/vnd.github.mercy-preview+json'},
+                success: (data) => {
+                    
+                    results.total = data.total_count
+                    results.items = data.items
+
+                    $('#total_count').html(data.total_count)
+                    $('#search').prop('style', '')
+                    $('#results').prop('style', '')
+                    $('#searching').prop('style', 'display:none')
+
+                    data.items.forEach(element => {
+
+                        let license = 'Unknown'
+                        if(element && element.license && element.license.name)
+                        {
+                            license = element.license.name
+                        }
+
+                        let updated = moment(element.updated_at || new Date()).format('YYYY-MM-DD')
```

Comment:
remove moment, 

get inspiration from https://stackoverflow.com/questions/23593052/format-javascript-date-to-yyyy-mm-dd

Reply:
done